tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -83,6 +83,15 @@ acl/test_acl_outer_vlan.py:
     conditions:
       - "'dualtor' in topo_name"
 
+#######################################
+#####          acstests              #####
+#######################################
+acstests:
+  skip:
+    reason: "It is not  tested for now"
+    conditions:
+      - True
```

Comment:
True is not correct condition, it will cause such problem:

```
INTERNALERROR>     if condition is None or condition.strip() == '':
INTERNALERROR> AttributeError: 'bool' object has no attribute 'strip'
```

Use "True" instead.